### PR TITLE
Pancakes Griddle

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Pancakes are now made in a different way. You make pancake batter with flour, milk, and eggs, then you pour it onto the griddle to make a raw pancake. After they cook for awhile you get a really nice 'cake brother.

while it is still cooking, turn it into blueberry or chocolate pancake by garnishing it on the griddle.

Sprites by tonight posted, maybe even a youtube video this is dope man

## Why It's Good For The Game

Pastry base is an affront to god, the Griddler demands it.

## Changelog
:cl:
add: Pancakes now griddle to be cooked
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/griddle.dm [hunk 1]
```diff
@@ -26,6 +26,20 @@
 	. = ..()
 	grill_loop = new(list(src), FALSE)
 	variant = rand(1,3)
+	RegisterSignal(src, COMSIG_ATOM_EXPOSE_REAGENT, .proc/on_expose_reagent)
+
+/obj/machinery/griddle/proc/on_expose_reagent(atom/parent_atom, datum/reagent/exposing_reagent, reac_volume)
+	SIGNAL_HANDLER
+
+	if(griddled_objects.len >= max_items || !istype(exposing_reagent, /datum/reagent/consumable/pancakebatter) || reac_volume < 5)
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/kitchen_machinery/griddle.dm:34]: :-1:
Make this a proc on the reagent itself, this behavior shouldn't be on the griddle.

Comment (Qustinnus) [reply code/modules/food_and_drinks/kitchen_machinery/griddle.dm:34]: adding a proc on reagents for each object a reagent might have interactions with is anything but modular. If we started making it possible to have like 10 reagents you can make items with on the griddle we can consider it, but there is no need for this to be on the reagent right now.

Comment (tralezab) [reply code/modules/food_and_drinks/kitchen_machinery/griddle.dm:34]: I talked with floyd about it, we agreed this wasn't big enough to add onto reagents

**Comment by Cyberboss** [comment]
┬─┬ ノ( ゜-゜ノ) 
let's talk

**Comment by TemporalOroboros** [comment]
Would it be possible to finagle the pancake batter proc into something that can put the other splashed reagents into the pancake batter?

**Comment by tralezab** [comment]
The whole "you can do anything" joke is a complete meme from goofkitchen, don't worry.

**Comment by tralezab** [comment]
Also: gotta remember to actually USE up the berries and chocolate in the recipe.

**Comment by wesoda25** [comment]
i think if you pour a lot of batter you should make a jumbo pancake @tralezab

**Review by Twaticus — APPROVED** [review]

**Pull request merged**